autoconfig.go

```diff
@@ -7,6 +7,10 @@ type AutoConfigVar struct {
 	Example     string        `json:"example,omitempty"`
 	Default     interface{}   `json:"default,omitempty"`
 	Enum        []interface{} `json:"enum,omitempty"`
+
+	// Exclude the value from being sent to the server. This essentially makes
+	// it a value which is only used in param templates.
+	Exclude bool `json:"exclude,omitempty"`
```

Comment:
Where does this actually get used?

Reply:
It gets used by Restish, see https://rest.sh/#/openapi?id=autoconfiguration. The Huma side is just putting it into the JSON so you don't see much code here for it.

Basically it lets you take in a param without also sending it to the server, which is important because e.g. Auth0 will return an error if you send extra params when requesting a token.